src/views/tutorial-view/components/next-previous/components/directional-link-box/directional-link-box.module.css

```diff
@@ -0,0 +1,101 @@
+.card {
```

Comment:
These styles are copied from Learn. Leaving them here in the stub as they may be a helpful reference during UI implementation.